Source/automap.cpp

```diff
@@ -17,6 +17,24 @@ int AutoMapYPos; // weak
 int AMPlayerX; // weak
 int AMPlayerY; // weak
 
+// color used to draw the player's arrow
+#define COLOR_PLAYER      (PAL8_ORANGE+1)
+// color for bright map lines (doors, stairs etc.)
+#define COLOR_BRIGHT      PAL8_YELLOW
+// color for dim map lines/dots
+#define COLOR_DIM         (PAL16_YELLOW+8)
+
+#define MAPFLAG_TYPE      0x000F
+// these are in the second byte

```

Comment:
The *type* is still in the first byte, only the *flag* is in the second byte, thus this comment should perhaps be moved to `MAPFLAG_HALFR`.

Reply:
All the following values are in the second byte. The type doesn't overlap with them.

Reply:
You are indeed correct.